scio-core/src/main/java/com/spotify/scio/options/ScioOptions.java

```diff
@@ -41,4 +42,9 @@
   @Description("Custom application arguments")
   String getAppArguments();
   void setAppArguments(String arguments);
+
+  @JsonIgnore
+  @Description("Config file with options")
```

Comment:
Make this clearer? Something like "JSON file to be parsed as PipelineOptions instead of command line arguments"?